orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/ClientRegistry.java

```diff
@@ -451,4 +453,18 @@ public ClientRegistry withClientLoCs(List<String> clientLoCs) {
         this.clientLoCs = clientLoCs;
         return this;
     }
+
+    @DynamoDbAttribute("PermitMissingNonce")
+    public boolean getPermitMissingNonce() {
+        return !identityVerificationSupported && permitMissingNonce;
+    }
+
```

Comment:
Can this be called `permitMissingNonce` as it returns a boolean?

This would read clearer at call sites e.g. `if (client.permitMissingNonce()) { ... }`

Reply:
Also, is this method not just for persistence in the record? We would return the field value rather than joining it with whether it's an identity RP, and have a different method for that.

Maybe the `permitMissingNonce` method should be the joined form, and the `getPermitMissingNonce` method should just be the raw value getter.